manila/tests/db/sqlalchemy/test_api.py

```diff
@@ -584,9 +584,18 @@ def test_share_get_all_by_export_location_not_exist(self, type):
     @ddt.unpack
     def test_share_get_all_with_count(self, filters, amount_of_shares,
                                       expected_shares_len):
-        [db_utils.create_share(display_name='fake_name_%s' % str(i))
-         for i in range(amount_of_shares)]
+        c_shares = [
+            db_utils.create_share(display_name='fake_name_%s' % str(i))
+            for i in range(amount_of_shares)]
 
+        share_server = db_utils.create_share_server()
+        [db_utils.create_share_replica(
+            replica_state=constants.REPLICA_STATE_ACTIVE,
```

Comment:
It looks a bit weird to have more than one active replica - but not a reason to reject the PR